packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -68,7 +68,16 @@
     "holdScreenLabel": "You're on hold",
     "openDtmfDialpadLabel": "Show dialpad",
     "dtmfDialpadPlaceholderText": "Enter number",
-    "outboundCallingNoticeString": "Calling..."
+    "outboundCallingNoticeString": "Calling...",
+    "participantJoinedNoticeString": "{displayNames} joined",
+    "twoParticipantJoinedNoticeString": "{displayName1} and {displayName2} have joined",
+    "threeParticipantJoinedNoticeString": "{displayName1}, {displayName2} and {displayName3} have joined",
+    "participantLeftNoticeString": "{displayNames} left",
+    "twoParticipantLeftNoticeString": "{displayName1} and {displayName2} have left",
+    "threeParticipantLeftNoticeString": "{displayName1}, {displayName2} and {displayName3} have left",
+    "unnamedParticipantChangedString": "unnamed participant",
+    "manyUnnamedParticipantsChanged": "unnamed participant and {numOfParticipants} other participants",
+    "manyParticipantsChanged": "{displayName1}, {displayName2}, {displayName3} and {numOfParticipants} other participants"
```

Comment:
Open to push back on this, I think it would be clearer to have manyParticipantsLeft and manyParticipantsJoined as its very difficult to know how `manyParticipantsChanged` would be used